controllers/automation/policyautomation_controller.go

```diff
@@ -89,6 +92,150 @@ func (r *PolicyAutomationReconciler) setOwnerReferences(
 	return nil
 }
 
+// getTargetListMap will convert slice targetList to map for search efficiency
+func getTargetListMap(targetList []string) map[string]bool {
+	targetListMap := map[string]bool{}
+	for _, target := range targetList {
+		targetListMap[target] = true
+	}
+
+	return targetListMap
+}
+
+// getClusterName will get the hub cluster name information
+func getClusterName() string {
+	// Get a config to talk to the apiserver
+	cfg := config.GetConfigOrDie()
+	hostStr := cfg.Host
```

Comment:
I don't think you need this separate variable. I think you could just always use `hubCluster`.